db/seeds.rb

```diff
@@ -51,32 +51,55 @@ def random_record_for_org(org, klass)
 # ----------------------------------------------------------------------------
 
 pdx_org = Organization.find_or_create_by!(short_name: "diaper_bank") do |organization|
-  organization.name    = "Pawnee Diaper Bank"
-  organization.street  = "P.O. Box 22613"
-  organization.city    = "Pawnee"
-  organization.state   = "Indiana"
+  organization.name = "Pawnee Diaper Bank"
+  organization.street = "P.O. Box 22613"
+  organization.city = "Pawnee"
+  organization.state = "Indiana"
   organization.zipcode = "12345"
-  organization.email   = "info@pawneediaper.org"
+  organization.email = "info@pawneediaper.org"
 end
 Organization.seed_items(pdx_org)
 
 sf_org = Organization.find_or_create_by!(short_name: "sf_bank") do |organization|
-  organization.name    = "SF Diaper Bank"
-  organization.street  = "P.O. Box 12345"
-  organization.city    = "San Francisco"
-  organization.state   = "CA"
+  organization.name = "SF Diaper Bank"
+  organization.street = "P.O. Box 12345"
+  organization.city = "San Francisco"
+  organization.state = "CA"
   organization.zipcode = "90210"
-  organization.email   = "info@sfdiaperbank.org"
+  organization.email = "info@sfdiaperbank.org"
 end
 Organization.seed_items(sf_org)
 
+sc_org = Organization.find_or_create_by!(short_name: "sc_bank") do |organization|
+  organization.name = "Second City Essentials Bank"
+  organization.street = Faker::Address.street_address
+  organization.city = Faker::Address.city
+  organization.state = Faker::Address.state_abbr
+  organization.zipcode = Faker::Address.zip_code
+  organization.email = "info@scdiaperbank.org"
+end
+Organization.seed_items(sc_org)
+
+# The list of organizations that will have donations, purchases, requests, distributions,
+# and the records those rely on generated.
+complete_orgs = [pdx_org, sc_org]
+
 # At least one of the items is marked as inactive
-Organization.all.each do |org|
+Organization.all.find_each do |org|
   org.items.order(created_at: :desc).last.update(active: false)
 end
 
+# Add a couple unique items based on random base items named after the sc_bank
+# so it will be clear if they are showing up where they aren't supposed to be
+4.times.each do |time|
+  sc_org.seed_random_item_with_name("Second City Item ##{time + 1}")
```

Comment:
nit: Not sure the value of abstracting this logic into a new method. We could just create four new items here and then call `sc_org.seed_items(second_city_only_items)`. And having all the logic in this file makes it easier to read imo.

But up to you.

Also another nit, I think the block parameter would normally be named `index` or `i` here. But doesn't really matter.

Reply:
Yes, `time` should absolutely be `index` as that convention is used elsewhere in the file.